### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by HalimaElbashier**
Fixed confirm button, added 'waiting' state and Arabic translation

### CODE REVIEW

File: account-entry-operations/models/account_move.py [hunk 1]
```diff
@@ -4,14 +4,8 @@
...
-    state = fields.Selection(selection=[
-        ('draft', _('Draft')),
-        ('waiting', _('Waiting')),
-        ('posted', _('Posted')),
-        ('cancel', _('cancelled')),
-    ], string=_('Status'), default='draft', required=True, readonly=True, copy=False, tracking=True)
-
-    wait = fields.Boolean(string='Waiting', default=False)
+    state = fields.Selection(selection_add=[
+        ('waiting', _('Waiting Approval')), ('posted',)], ondelete={'waiting': 'cascade'})
```
Comment (bashierelbashier) [inline account-entry-operations/models/account_move.py:8]: Please change ondelete from cascade to "set default"

File: account-entry-operations/i18n/ar_001 (1).po
Comment (bashierelbashier) [inline account-entry-operations/i18n/ar_001 (1).po:1]: The name of this file should not include (1)
It has to be ar_001.po

File: account-entry-operations/models/account_move.py (hunk 1 above)
Comment (HalimaElbashier) [reply account-entry-operations/models/account_move.py]: Changed to 'set default'

**Review by bashierelbashier — APPROVED** [review]
Approved

**Pull request merged**